Resources/ChromeToBrave/ChromeToBrave.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/local/bin/python3
+# -*- coding: utf-8 -*-
+
+import os
+import subprocess
+import sys
+import plistlib
+import re
+
+chrome_manifest_domain = "com.google.Chrome"
+brave_manifest_domain = "com.brave.Browser"
+manifests_subfolder = os.path.join( "Manifests", "ManagedPreferencesApplications" )
```

Comment:
This assumes that the current working directory is the ProfileManifests folder.  If that's not the case, I'd love some output to indicate this fact.